src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -49,18 +49,51 @@
       <DataSource Persistence="ProjectFile" PersistedName="Win32Resource" />
     </StringProperty.DataSource>
   </StringProperty>
-  <StringListProperty Name="AvailablePlatforms" Separator="," Visible="False" />
-  <StringProperty Name="FileName" DisplayName="Project File" ReadOnly="True">
+    <StringProperty Name="DefineConstants" DisplayName="Define Constants"/>
+    <EnumProperty Name="PlatformTarget" DisplayName="Platform Target"/>
```

Comment:
I can add validation similar to [this ](https://github.com/jinujoseph/roslyn-project-system/blob/962c8c7a4537b085f070b543893996a6e1b66bdf/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/BuildPropertyPage/PlatformTargetValueProvider.cs), but not sure if it's useful if the values are not driven from the xaml.